### ISSUE TITLE
Add autocomplete widgets to /admin/harvest/property/ form

### ISSUE THREAD

**Issue by tristanlatr** [LesFruitsDefendus/saskatoon-ng#260]
More specifically in the edit view of Properties: we'd like to have an Actor autocomplete owner drop down.

**Comment by sophiessa** [comment]
I can see an autocomplete drop down, is the issue solved. If not solved yet, could you clarify it?

**Comment by tristanlatr** [comment]
Yes there are drop down in the front end, edits property page. But there is no autocomplete in the admin panel. This issue is about the admin panel.

hope it’s more clear now.

**Comment by mina-andajos** [comment]
Is this the dropdown you mean?

![image](https://user-images.githubusercontent.com/52796958/169353863-4e990079-dc23-4329-9b0d-82a594c00693.png)

**Comment by tristanlatr** [comment]
Hello @mina-andajos,

Yes it's exactly this one,

Talk to you later,

**Comment by Superpat** [comment]
As far as I can see this seems completed.
